### PR TITLE
OTG Test - DP-1.11: Bursty traffic test

### LINKED ISSUE #1209: OTG: DP-1.11: Bursty traffic test

### PR DESCRIPTION
**Pull request by octpetre**
OTG conversion for **_DP-1.11: Bursty traffic test_**
Resolves #1209

### CODE REVIEW

**Comment by octpetre** [comment]
Waiting for merger of #1351

File: feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go [hunk 1]
```diff
@@ -0,0 +1,1269 @@
...
+		}
+		output := i.GetOrCreateOutput()
+		schedulerPolicy := output.GetOrCreateSchedulerPolicy()
+		schedulerPolicy.SetName(tc.scheduler)
+		queue := output.GetOrCreateQueue(tc.queueName)
+		queue.SetName(tc.queueName)
+		gnmi.Replace(t, dut, gnmi.OC().Qos().Config(), q)
+	}
+}
+func ConfigureCiscoQos(t *testing.T, dut *ondatra.DUTDevice) {
```
Comment (jasdeep-hundal) [inline feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go:954]: Empty line before the previous function + t.Helper here as well.

File: feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go [hunk 2]
```diff
@@ -0,0 +1,1269 @@
...
+			t.Logf("Running traffic 1 on DUT interfaces: %s => %s ", dp1.Name(), dp3.Name())
+			t.Logf("Running traffic 2 on DUT interfaces: %s => %s ", dp2.Name(), dp3.Name())
+			t.Logf("Sending traffic flows: \n%v\n\n", trafficFlows)
+			ate.OTG().StartTraffic(t)
+			time.Sleep(30 * time.Second)
+			ate.OTG().StopTraffic(t)
+			time.Sleep(30 * time.Second)
+
+			for trafficID, data := range trafficFlows {
+				ateTxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(trafficID).Counters().OutPkts().State())
```
Comment (jasdeep-hundal) [inline feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go:453]: Change to a single query for both similar to: https://github.com/openconfig/featureprofiles/blob/e8068e3f70aeb6cbcbd454bd72337106e76c473c/feature/gribi/otg_tests/supervisor_failure_test/supervisor_failure_test.go#L163

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go (hunk 2 above)
Comment (octpetre) [reply feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go:453]: Done

File: feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go (hunk 1 above)
Comment (octpetre) [reply feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go:954]: Done

File: feature/qos/otg_tests/bursty_traffic_test/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+*   Traffic types:
+
+    *   All the traffic tests apply to both IPv4 and IPv6 traffic.
+
+*   Queue types:
+
+    *   NC1 will have strict priority queues
+        *   AF4/AF3/AF2/AF1/BE1/BE0 will use WRR queues.
+    *   NC1 and AF4 will have strict priority queues with NC1 having higher
```
[the remark this answers is not in the document]
Comment (octpetre) [reply feature/qos/otg_tests/bursty_traffic_test/README.md:35]: Done

**Comment by jasdeep-hundal** [comment]
@octpetre : Requesting one more merge with the latest code to pick up changes in https://github.com/openconfig/featureprofiles/commit/5e779e86a219b3ef5b5be8f28ac5e0a536d5f81e.

**Review by jasdeep-hundal — APPROVED** [review]

**Pull request merged**